### DISCUSSION TITLE
portability recommendations

### DISCUSSION THREAD

**Discussion by klausler** [codee-com/fortran-modernization#2] [General]
I've written up a bunch of Fortran portability recommendations at https://github.com/klausler/fortran-wringer-tests?tab=readme-ov-file#writing-portable-fortran -- please feel free to link to that page or take ideas from it.

**Comment by alvrogd** [comment]
Thank you for sharing your portability recommendations. They definitely look like a valuable resource! As portability is indeed a main concern when writing modern Fortran, we'll take note to review your notes thoroughly and will certainly consider integrating your insights, ensuring proper credit is given where due :)

Naturally, if you have specific improvements in mind for any part of the recommendations and are interested, please feel free to open a pull request so we can review them together. Alternatively, we can also continue the conversation here for further discussion!

Thank you once again for your contributions!
